### ISSUE TITLE
Failed to resize volume for multitude of reasons

### ISSUE THREAD

**Issue by NotANormalNerd** [hetznercloud/csi-driver#96]
Hi ^^, we are running `Kubernetes Version: v1.16.3` with the current csi-driver 1.2.2 and we had some problems resizing the persistentVolumeClaim. We edited the persistentVolumeClaim and changed the size request from 10Gi to 25Gi, then happened nothing.

I checked the logs of the resizer and found: 
```
E0115 15:29:31.429239 1 controller.go:319] Mark pvc "postgresql-server/postgresql-data-postgresql-0" as resizing failed: can't patch status of PVC postgresql-server/postgresql-data-postgresql-0 with persistentvolumeclaims "postgresql-data-postgresql-0" is forbidden: User "system:serviceaccount:kube-system:hcloud-csi" cannot patch resource "persistentvolumeclaims/status" in API group "" in the namespace "postgresql-server"
```
which I added manually by editing the ClusterRole hcloud-csi, but then I encountered:
```
E0115 15:52:32.208845 1 controller.go:366] Update capacity of PV "pvc-4deaa3cc-6114-491e-99df-c15598377917" to 25Gi failed: update capacity of PV pvc-4deaa3cc-6114-491e-99df-c15598377917 failed: persistentvolumes "pvc-4deaa3cc-6114-491e-99df-c15598377917" is forbidden: User "system:serviceaccount:kube-system:hcloud-csi" cannot patch resource "persistentvolumes" in API group "" at the cluster scope
```
which I also manually added by editing the ClusterRole

The Claim and the Volume got resized and I deleted/restarted the pod which got stuck initializing. Looking at the node-drive I found:
```
level=error ts=2020-01-15T15:55:55.622000442Z component=grpc-server msg="handler failed" err="rpc error: code = Internal desc = failed to resize volume: resize of device /dev/disk/by-id/scsi-0HC_Volume_3988862 failed: executable file not found in $PATH. resize2fs output: "
```
Which I fixed by hopping into the shell and `apk add e2fsprogs-extra`
After that the pod came up fine and resized.

I can't really say if this is a legit bug or if this is on our end, since we are currently toying around on a cluster.

Maybe somebody can try to reproduce this?

Thanks ^^

**Comment by Dylan-DutchAndBold** [comment]
Sadly running into the same issue here with v1.2.1

    Mark pvc "default/masked-name" as resizing failed: can't patch status of PVC default/masked-name with persistentvolumeclaims "masked-name" is forbidden: User "system:serviceaccount:kube-system:hcloud-csi" cannot patch resource "persistentvolumeclaims/status" in API group "" in the namespace "default"

**Comment by JannikZed** [comment]
I'm facing the same issue. First some missing privileges and afterwards the missing $PATH as @NotANormalNerd already described. -- Seems like just very small things that can be fixed easily ..

**Comment by LKaemmerling** [comment]
We just released v1.2.3 which should fix this issue.

**Comment by vitobotta** [comment]
I referenced another issue but I think it's better to continue here since I am having these same issues with 1.2.3.  I fixed by making the changes to the cluster role described here https://github.com/hetznercloud/csi-driver/pull/97/commits/d5762423488d1d73d37184b6a313311bbe61bfcd  - so it looks like that change didn't make it into 1.2.3? Thanks
